src/lib.rs

```diff
@@ -9,6 +9,26 @@
 //!   structures.
 //!
 
+// Declares a 'const' member of a test module that's a new ExtensionId.
+// field_name should be an UPPERCASE i.e. name of a const.
+// This is here so that it is usable throughout the crate, but is not
+// visible from outside (as it would be if we used macro_export)
+// - as it won't *work* from outside because of the call to new_unchecked.
+#[allow(unused_macros)] // Because not used *in this file*
+macro_rules! test_const_ext_id {
```

Comment:
- Either put it in `extensions.rs` or in `macros.rs`.

- Use `$crate` to refer to `hugr` in the `use` statements (here it doesn't change much, but if used from another crate `crate::*` refers to the user's crate).

Reply:
Can this be referred to from outside the crate? It's not `#[macro_export]` so I didn't think it could be. It'd be useless ATM if it could be referred to, as it generates calls to the `pub(crate) fn new_unchecked`...